contributions/catalog/webapps/templates/contribute/talent_details.html

```diff
@@ -125,7 +125,7 @@ <h4>Minimum user privacy level</h4>
         {% if post.minEndUserRoles %}
         <div class="row">
             <div class="col-12 col-md-3">
-                <h4>Minimum set of end-user roles</h4>
+                <h4>Minimum Set of End-user Roles</h4>
```

Comment:
```suggestion
                <h4>End-user Roles</h4>
```

I think, here, "Minimum Set" could be omitted.